gammapy/utils/interpolation.py

```diff
@@ -79,16 +79,17 @@ def __init__(
                 points_scaled[0], values_scaled, axis=axis
             )
 
-    def __call__(self, points, method="linear", clip=True, **kwargs):
+    def __call__(self, points, method=None, clip=True, **kwargs):
         """Interpolate data points.
 
         Parameters
         ----------
         points : tuple of `~numpy.ndarray` or `~astropy.units.Quantity`
             Tuple of coordinate arrays of the form (x_1, x_2, x_3, ...). Arrays are
             broadcasted internally.
-        method : {"linear", "nearest"}
-            Linear or nearest neighbour interpolation.
+        method : {None, "linear", "nearest"}
+            Linear or nearest neighbour interpolation. None will choose the default
+            defined on init.
```

Comment:
Maybe this could be made more explicit since `method` is not an explicit argument of `ScaledRegularGridInterpolator.__init__()`

Reply:
I extended the docstring.